docs/applications/index.rst

```diff
@@ -6,35 +6,39 @@ Applications are individual *atomic* services that are configured and deployed t
 Each environment can opt whether to deploy an application, and also customize the configuration of the application.
 This section of the documentation describes each Phalanx application.
 
+The category names correspond to the project name in phalanx/applications/_project_.
+Infrastructure is specifically reserved for applications that are necessary for "proof of life" deployments, eg by non-Rubin deployers who are only interested in phalanx as an internal developer environment.
+
 To learn how to develop applications for Phalanx, see the :doc:`/developers/index` section.
 
 .. toctree::
    :maxdepth: 1
-   :caption: Cluster infrastructure
+   :caption: infrastructure ("bare" Phalanx infrastructure)
```

Comment:
I'm a little worried this caption is too long. `infrastructure` is probably ideal. I'd have to see a preview build.